src/promptflow/promptflow/_sdk/_service/apis/line_run.py

```diff
@@ -90,4 +76,16 @@ def get(self):
         line_runs = client._traces.list_line_runs(
             session_id=args.session_id,
         )
-        return [asdict(line_run) for line_run in line_runs]
+
+        line_runs_id_list, line_runs_dict = [], []
+        for line_run in line_runs:
+            line_runs_id_list.append(line_run.line_run_id)
+            line_runs_dict.append(asdict(line_run))
+
+        # create a hash of the sorted list of line run ids
+        # and add it to the response header
+        # so that UX can use it to determine if the line runs have changed
+        line_runs_id_list.sort()
+        line_runs_hash = hashlib.sha256(",".join(line_runs_id_list).encode()).hexdigest()
+
+        return line_runs_dict, 200, {HASH_HEADER_IN_LINE_RUNS_LIST: line_runs_hash}
```

Comment:
why do we need to do this in server side for UX?

Reply:
it's not a very heavy logic for server side, so implement here; I might need to sync with Kaiyi on this.

Reply:
also, we have full knowledge on whether a hash should change - one advantage from our side.

Reply:
synced with Kaiyi, will remove this and client will do the hash calculation